contracts/contracts/ERC677/README.md

```diff
@@ -0,0 +1,12 @@
+# Chainlink ERC677
+
+## starkgate_token.cairo
+
+Similare to the starkgate ERC20 but with some added functionality from ERC677.
+
+- transferAndCall -> Transfers tokens to receiver, via ERC20's transfer(address,uint256) function. It then logs an event Transfer(address,address,uint256,bytes). Once the transfer has succeeded and the event is logged, the token calls onTokenTransfer(selector, data_len, data) on the receiver with the function's selector, and all the parameters required by the function that you want to call next.
+
+## linkReceiver.cairo
```

Comment:
The receiver should not be link specific in any way, we are implementing the ERC677 protocol here so a contract like this should be named something like Mock_ERC677Receiver, as this is used for testing, right?

Reply:
Yes, I call it like this because of this https://github.com/smartcontractkit/LinkToken/tree/f307ea6d4c02dd87ea4c1f1032e3e384ceac20c2/contracts/v0.6/mocks